src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -303,6 +310,47 @@ def get_sonic_version_file():
 
     return SONIC_VERSION_YAML_PATH
 
+
+# Get hardware information
+def get_platform_info():
```

Comment:
Just thinking that we have `get_platform()` and `get_platform_info()`. Hard to discern the difference by name. If we can't think of a more descriptive name, we can merge this.

Reply:
I'm not sure there is a more descriptive method name possible given the purpose of the function is to provide various platform information that we have deemed "good to know" to the user. 

Alternates could be `get_platform_summary()` or `get_hardware_facts()` 

But unless you *love* any of these suggestions compared to what we have I suggest we keep with the current one as I don't see a large difference between these suggestions other than personal taste. 

Open to suggestions as well.

Reply:
I agree it's hard to think of much better. We can move ahead with this.